MANIFEST.in

```diff
@@ -1,3 +1,3 @@
-include LICENSE README.md
```

Comment:
`MANIFEST.in`

Reply:
The SDist workflow was erroring out due to this -
```
check-manifest...........................................................Failed
- hook id: check-manifest
- exit code: 1
lists of files in version control and sdist do not match!
missing from sdist:
  noxfile.py
suggested MANIFEST.in rules:
  include *.py
```
https://github.com/scikit-hep/vector/runs/7136492072?check_suite_focus=true

Is there a way to ignore `noxfile`?

Reply:
Thank you!